packages/@uppy/aws-s3-multipart/types/index.d.ts

```diff
@@ -33,29 +34,24 @@ export interface AwsS3MultipartOptions extends PluginOptions {
     ) => MaybePromise<{ uploadId: string; key: string }>
     listParts?: (
       file: UppyFile,
-      opts: { uploadId: string; key: string; signal: AbortSignal }
+      opts: { uploadId: string; key: string }
     ) => MaybePromise<AwsS3Part[]>
-    getTemporarySecurityCredentials?: boolean | (() => MaybePromise<AwsS3STSResponse>)
     signPart?: (
       file: UppyFile,
-      opts: { uploadId: string; key: string; partNumber: number; body: Blob, signal: AbortSignal }
+      opts: { uploadId: string; key: string; partNumber: number; body: Blob; signal: AbortSignal }
```

Comment:
I tried to match the style as it was done; however this appears to be an inconsistency so I did change it here.